### PR TITLE
Adding WriteCachedMongoPlatformRepository + java opts to avoid "OutOfMemoryError: GC overhead limit exceeded"

### PR DESCRIPTION
**Pull request by Lucas-C**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <!-- Dependencies -->
         <axon.version>3.3.7</axon.version>
-        <hesperides.version>4.2.0-SNAPSHOT</hesperides.version>
+        <hesperides.version>4.2.1-SNAPSHOT</hesperides.version>
```
Comment (thomaslhostis) [inline pom.xml]: On est en 4.2.2-SNAPSHOT :)

File: src/main/java/org/hesperides/datamigration/run/processing/AbstractProcessingService.java [hunk 2]
```diff
@@ -94,8 +94,32 @@ public void process(long fromTimestamp, long toTimestamp) {
                     .stream()
                     .map(b -> new String(b, UTF_8))
                     //                .skip(0).limit(1500)
-//                    .filter(s -> s.equalsIgnoreCase("module-bus-front-1.0.0-SNAPSHOT-wc"))
-//                              || s.equalsIgnoreCase("module-ivs-batch-4.0.9.02-wc"))
+                    .filter(s -> s.equalsIgnoreCase("platform-VSL-INT3")
+                              || s.equalsIgnoreCase("module-wdiWatts-war-20.0.194.0-release")
+                              || s.equalsIgnoreCase("module-mailConsumer-52.00.0.0-release")
+                              || s.equalsIgnoreCase("module-vscAdmin-war-45.00.0-0-release")
```
Comment (thomaslhostis) [inline src/main/java/org/hesperides/datamigration/run/processing/AbstractProcessingService.java]: À commenter ?

File: src/main/java/org/hesperides/datamigration/run/processing/AbstractProcessingService.java [hunk 3]
```diff
@@ -198,7 +222,6 @@ private boolean containsDeletedEventThatIsNotTheLastEvent(List<LegacyEventWrappe
                     .map(legacyEventSerializer::deserialize)
                     .filter(legacyEvent -> legacyEvent.getTimestamp() > fromTimestamp && legacyEvent.getTimestamp() < toTimestamp)
                     .collect(Collectors.toList());
-        });
-        return status.getResult();
+        }).getResult();
```
Comment (thomaslhostis) [inline src/main/java/org/hesperides/datamigration/run/processing/AbstractProcessingService.java]: 👍

File: src/main/java/org/hesperides/datamigration/uglyhack/WriteCachedMongoPlatformRepository.java [hunk 4]
```diff
@@ -0,0 +1,210 @@
+package org.hesperides.datamigration.uglyhack;
```
Comment (thomaslhostis) [inline src/main/java/org/hesperides/datamigration/uglyhack/WriteCachedMongoPlatformRepository.java]: 👍

File: pom.xml (hunk 1 above)
Comment (Lucas-C) [reply pom.xml]: ok, corrigé

File: src/main/java/org/hesperides/datamigration/run/processing/AbstractProcessingService.java (hunk 2 above)
Comment (Lucas-C) [reply src/main/java/org/hesperides/datamigration/run/processing/AbstractProcessingService.java]: oui, merci

**Pull request merged**